login-script.js

```diff
@@ -1,13 +1,15 @@
+//Function used to initialize Firebase UI components (login with google)
```

Comment:
Please adopt a consistent style. I recommend [Google JavaScript Style Guide](https://google.github.io/styleguide/jsguide.html). You ought to be able to find some kind of automatic format checker.

In particular (here and elsewhere):

```
// Function used to initialize Firebase UI components (login with google).
```

i.e. comments are sentences, separated by 1 space from the `//`.